src/docs/components/login-form/LoginFormDoc.js

```diff
@@ -49,7 +49,7 @@ export default class LoginFormDoc extends Component {
             <dt><code>forgotPassword  {"{element}"}</code></dt>
             <dd>A link that would take the user to a new page.</dd>
             <dt><code>logo            {"{element}"}</code></dt>
-            <dd>A logo component.</dd>
+            <dd>A react node. Best suited to an svg element or font icon.</dd>
```

Comment:
can we remove the font icon? I don't think this is something we want to recommend. Also would you mind fixing the conflicts?

Sorry for taking such a long time to get back to you on this.